rules/cloud/azure/azure_privileged_account_no_saw_paw.yml

```diff
@@ -0,0 +1,30 @@
+title: Privilege Role Elevation Not Occuring on SAW or PAW
+id: 38a5e67b-436a-4e77-9f73-f48a82626890
+status: experimental
+description: Detects when an account fails a sign-in when in from a PAW or SAW device
+author: Mark Morowczynski '@markmorow', Yochana Henderson, '@Yochana-H'
+date: 2022/08/11
+references:
+    - https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/security-operations-privileged-accounts#things-to-monitor
+logsource:
+    product: azure
+    service: signinlogs
+detection:
+    selection:
+        properties.message|contains|all:
+            - Add memmber to role completed (PIM aciviation)
```

Comment:
Single value elements should be in the same line. So it should be like `properties.message|contains: 'Add memmber to role completed (PIM aciviation)'`

Also the `all` modifier can't be used for single values list (at least 2 to use it)

Reply:
Updated